model/src/main/java/org/cloudfoundry/identity/uaa/zone/OrchIdentityZoneEntity.java

```diff
@@ -0,0 +1,19 @@
+package org.cloudfoundry.identity.uaa.zone;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class OrchIdentityZoneEntity {
```

Comment:
Now I am thinking, we can use OrchestratorZoneEntity and in row mapper ignore the additional fields right, instead of creating new one.

Reply:
Yes that can be one option but that will be an incorrect representation of the table structure. Usually the entity should be a representation of the table.

Reply:
Implemented as suggested